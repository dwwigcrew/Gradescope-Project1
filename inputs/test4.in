4
1
1048
10
7
698