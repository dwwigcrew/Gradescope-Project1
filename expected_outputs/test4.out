
Welcome to the ENEE140 pseudo-random number generator!
1: Print RND_MAX
2: Generate uniformly-distributed positive integers
3: Generate uniformly-distributed positive integers, up to a given limit
4: Generate uniformly-distributed integers, from a given range
5: Generate uniformly-distributed floats, from a given range
6: Generate exponentially-distributed floats

Enter your choice (1-6):
Select the algorithm to use.

Enter your choice (1-3):
Select the seed for the random number generator:
How many numbers should I generate:
Enter the minimum number to generate:
Enter the maximum number to generate: 43 696 348 586 406 679 628 478 490 112
